cinema_service/settings.py

```diff
@@ -43,7 +43,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "rest_framework",
-    "rest_framework.authtoken",
```

Comment:
Why did you remove this? Can you explain, please?

Reply:
@IvanGLS ?

Reply:
@IvanGLS please explain the reason for removal this